SmartDeviceLink/SDLLifecycleManager.m

```diff
@@ -161,14 +161,16 @@ - (void)sdl_startWithReadyHandler:(SDLManagerReadyBlock)readyHandler {
 }
 
 - (void)stop {
```

Comment:
Why is this method being called at all in this situation? Your issue doesn't make it clear why `stop` would be called.

Reply:
Some apps are checking the firmware version of the External Accessory that is connected to make sure they want the app to show on that specific hardware (ex.  Navigation apps may not want to be shown on SYNC 1.1 but do want to be shown on SYNC 3).  So they are observing the `EAAccessoryDidConnectNotification` and `EAAccessoryDidDisconnectNotification` themselves and calling `start` after checking the EAAccessory `firmwareRevision` property of `EAAccessory`.  They call `stop` on `EAAccessoryDidDisconnectNotification`.  Even if an app calls `stop` in `managerDidDisconnect` delegate method we see this issue.